migration/db/schema.rb

```diff
@@ -144,4 +144,26 @@
     t.index ["hash"], name: "index_transactions_on_hash", unique: true
   end
 
+  create_table "uncle_headers", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
```

Comment:
this table should also index on block_number and be deleted during reorg.
but i am not sure we need this table in the first place. i thought the info in table `headers` was enough...